src/index.html

```diff
@@ -142,10 +206,33 @@
   </head>
 
   <body>
-    <div class="header-logo-container logo-position logo-position-start">
+    <div class="logo-container logo-position logo-position-start">
```

Comment:
@Wirnhier this conflict was likely not resolved correctly. It should be `header-logo-container` like on `development`.